os/board/rtl8721csm/src/component/os/tizenrt/rtk_netmgr.c

```diff
@@ -318,6 +319,19 @@ trwifi_result_e wifi_netmgr_utils_scan_ap(struct netdev *dev, trwifi_ap_config_s
 	return wuret;
 }
 
+trwifi_result_e wifi_netmgr_utils_single_channel_scan(struct netdev *dev, trwifi_scan_config_s *scan_config)
+{
+	if (!scan_config) {
+		return TRWIFI_INVALID_ARGS;
+	}
+
+	if (cmd_wifi_scan_with_ssid(scan_config) != RTW_SUCCESS) {

```

Comment:
You compared return value with RTW_SUCCESS but cmd_wifi_scan_with_ssid you imeplemented returns 0 on success.
I think it is better to change 0 to RTW_SUCCESS in cmd_wifi_scan_with_ssid and return type of cmd_wifi_scan_with_ssid 
 from int8_t to rtw_result_t.